src/main/resources/db/changelog/changes/v1.1.0/sql/create-mat-view-pol-payment-status.sql

```diff
@@ -1,5 +1,19 @@
-CREATE MATERIALIZED VIEW IF NOT EXISTS drv_pol_payment_status AS
-SELECT DISTINCT jsonb ->> 'paymentStatus' AS payment_status
-FROM ${tenant_id}_mod_orders_storage.po_line;
+DROP MATERIALIZED VIEW IF EXISTS drv_pol_payment_status;
 
-REFRESH MATERIALIZED VIEW drv_pol_payment_status;
+CREATE MATERIALIZED VIEW drv_pol_payment_status AS
```

Comment:
Does this need added to the refresh materialized view service (or wherever we have the list of materialized views hard-coded)

Reply:
Yes, it's already there as part of my previous ticket (it's just that, because I didn't have a unique index on it originally, the refresh was failing and the API was throwing a 500 error)